packages/app/src/containers/SearchConditions/selectedCondition.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import {
+    Text,
+    View,
+    StyleSheet,
+} from 'react-native';
+import { Button, CheckBox, Icon } from "react-native-elements";
+import { Header } from "../../components";
+interface SearchConditionsState {
+    no: boolean;
+}
+
+interface OwnProps {
+    navigation: any;
+    route: any;
+}
+
+type Props = OwnProps;
+
+class SelectedConditionPage extends React.Component<Props, SearchConditionsState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            no: true,
+        };
+    }
+
+
+    public componentDidMount() {
+    }
+
+    public render() {
+        const Conditions = this.props.route.params.condition && this.props.route.params.condition[0];
+        return (
+            <React.Fragment>
+                <Header
+                    title={'CONDITIONS'}
+                    leftIcon={true}
+                    rightIcon={true}
+                    navigation={this.props.navigation} />
+                <View style={styles.container}>
+                    <View style={styles.topView}>
+                        <Text style={styles.heading}>{Conditions}</Text>
+                        <Text style={styles.subheading}>{"This condition has been added to your journal. Do you suufer from any other conditions?"}</Text>
```

Comment:
'suffer' is misspelled

Reply:
@petermsouzajr Ok I will fix this typo error